packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -260,7 +261,7 @@ export class FluidDevtools implements IFluidDevtools {
 		if (FluidDevtools.I !== undefined) {
 			console.warn(
 				"Devtools have already been initialized. " +
-					"Existing Devtools must be closed (see closeDevtools) before new ones may be initialized. " +
+					"Existing Devtools must be closed (see DevtoolsDisposed) before new ones may be initialized. " +
```

Comment:
Ooh, this comment is stale. We should probably update this to say something like:
```suggestion
					"Existing Devtools instance must be disposed before new ones may be initialized. " +
```